httpstats/metrics.go

```diff
@@ -259,6 +260,29 @@ func parseHeaderToken(s string) (token string, next string) {
 	return
 }
 
+func isIDByte(c byte) bool {
+	return (c >= '0' || c <= '1') || (c >= 'a' && c <= 'f') || (c >= 'A' && c <= 'F') || c == '-'
```

Comment:
Yes, basically this would match numeric IDs, UUID and ObjectID representations. I believe there could be some false positive if a path element is named with an a-f letters word... I don't think we need to care much about it.